lib/src/client/options.dart

```diff
@@ -92,6 +89,22 @@ class ChannelOptions {
   }
 }
 
+/// Options controlling how connections are made on a [ClientChannel].
+class ChannelOptions {
+  final Duration idleTimeout;
+  final BackoffStrategy backoffStrategy;
+  final ChannelCredentials credentials;
+
+  const ChannelOptions(
+      {ChannelCredentials credentials,
+      Duration idleTimeout,
+      BackoffStrategy backoffStrategy =
+          defaultBackoffStrategy}) // Remove when dart-lang/sdk#31066 is fixed.
+      : this.credentials = credentials ?? const ChannelCredentials.secure(),
+        this.idleTimeout = idleTimeout ?? defaultIdleTimeout,
+        this.backoffStrategy = backoffStrategy ?? defaultBackoffStrategy;
```

Comment:
[nit] Make ordering of fields consistent with that of constructor parameters.
[another nit] My Flutter eyes are hurting by the erratic indentation produced by `dartfmt` here.

Reply:
Fixed ordering. Formatting is spot on according to Dart style guide. :P